src/routes.json

```diff
@@ -212,6 +212,13 @@
         "view": "privacypolicy/privacypolicy",
         "title": "Privacy Policy"
     },
+    {
+        "name": "research",
+        "pattern": "^/research/?$",
+        "routeAlias": "/research",
+        "view": "research/research",
+        "title": "Research"
+    },
```

Comment:
It's looking good. Thanks for remembering the route. We'll also need a `research-redirect` route for  `/info/research`. It should be just after `/info-redirect` at line 346.  I'm also going to pull this PR down and check it locally. The redirect pattern should look like the `donate-redirect` one, but the alias should just be `/info/research` Thanks for doing this!

Reply:
I remembered the route because I couldn't see the results otherwise :stuck_out_tongue: 
So, just add this in the right place?
```json
{
    "name": "research-redirect",
    "pattern": "^/info/research/?$",
    "routeAlias": "/info/research",
    "redirect": "/research"
}
```

Reply:
Yes that looks good! No worries on the milestone - we just forgot to move the PR to the next one :)

Reply:
As people on the Wiki would say: {{done}}